scripts/setup_network_storage.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python3
+
+# Copyright (C) SchedMD LLC.
```

Comment:
Google it

Reply:
Updated. It's just other script's licenses are still not updated.

Reply:
Closing in favor of my updated comment. I should have just posted it here, whoops.